src/Sets/VPolytope.jl

```diff
@@ -102,7 +102,7 @@ If it is empty, the result is ``-1``.
 julia> v = VPolytope();
 
 julia> v.vertices
-0-element Array{Array{Float64,1},1}
+Array{Float64,1}[]
```

Comment:
This change only works in v1.5 but not in the older versions. I suggest to just remove the output. Alternatively we can do `isempty(v.vertices)` or something.